docs/en/cloud/marketplace.md

```diff
@@ -4,20 +4,21 @@ slug: /en/cloud/marketplace
 title: CSP Marketplace
 ---
 
-## How can I start using ClickHouse Cloud connected to my AWS/GCP/Azure account billing?
+## Sign up for ClickHouse Cloud connected to your Cloud Service Provider (CSP) - AWS/GCP/Azure - billing account
```

Comment:
This is a very long title. Can we turn it into a paragraph instead?

```suggestion
This page covers how to sign up for ClickHouse Cloud, that is connected to a cloud service provider, such as AWS, GCP, or Azure.
```

We'll also have to update the header levels to `##` if we're changing this title.